packaging/rpm/Makefile

```
@@ -30,8 +33,10 @@ srpm: build_prepare
 		-r $(MOCK_CONFIG) \
```

```suggestion
		-r $(MOCK_CONFIG) \
		$(MOCK_OPTIONS) \
```

Lets go with a method of specifying any other options.